MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs

```diff
@@ -279,6 +304,37 @@ private string GetEncoderPathFromDirectory(string path, string filename, bool re
             }
         }
 
+        private Version GetLinuxKernelVersion()
```

Comment:
You should be able to use `Environment.OSVersion.Version`

Reply:
Oh, thanks. I actually missed such an easy method. 🤣